src/layouts/homepage.html.jade

```diff
@@ -34,19 +34,20 @@ html(lang="en")
                         span.title Cozy&nbsp;
                         | a Personal Cloud
                         br
-                        | You can Host, Hack
-                        | and Delete
-                    h4
-                        | Your contacts, calendars and files are stored on your
-                        | hardware and synced with your devices
+                        | you can host, customize
+                        | and fully control
```

Comment:
My bad, my point was about the sentence:

"Your contacts, calendars and files are stored on your hardware..."

It's super explicit and required by people to make sense about what is Cozy. I thought it was dropped in a lesser paragraph. But it looks like things it didn't change much.

About the tagline, I agree to remove negative words from it.